src/app/app.component.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { TestBed, async } from '@angular/core/testing';
+import { TestBed, waitForAsync } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
-  beforeEach(async(() => {
+  beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
       imports: [
         RouterTestingModule
```

Comment:
It's better to import modules of components used in the `AppComponent` in the test module.
```suggestion
        RouterTestingModule,
        NbThemeModule.forRoot(),
        NbLayoutModule,
        NbActionsModule,
        NbMenuModule.forRoot(),
        NbContextMenuModule,
```